### PR TITLE
Update Commonvoice Recipe with Conformer Settings

### PR DESCRIPTION
**Pull request by ftshijt**
How about just pack the best model for each lang? I don't think we should provide all the models for every experiment.

### CODE REVIEW

File: egs/commonvoice/asr1/conf/tuning/train_conformer_standard.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# network architecture
```
Comment (sw005320) [inline egs/commonvoice/asr1/conf/tuning/train_conformer_standard.yaml:1]: what does "standard" mean?
If this means Pencheng's standard setup, I think you can just say `train_conformer.yaml`

**Review by sw005320 — COMMENTED** [review]
Yes, you can just pack the best model for each language.
Please prepare them and reflect them on `RESULTS.md` accoridngly.

**Comment by ftshijt** [comment]
It's ready for merge.

**Comment by sw005320** [comment]
why does https://github.com/espnet/espnet/blob/5db9f89fd7bc2d48e979c885fd5b29edabfcbc6f/egs/commonvoice/asr1/RESULTS.md#train_es_pytorch_conformer only have the test WER?

**Comment by ftshijt** [comment]
> why does https://github.com/espnet/espnet/blob/5db9f89fd7bc2d48e979c885fd5b29edabfcbc6f/egs/commonvoice/asr1/RESULTS.md#train_es_pytorch_conformer only have the test WER?

It's weird. I found I made a key-interrupt at the time for this WER computing. I just reran it and added it into the RESULT.md

**Comment by sw005320** [comment]
Many many thanks @ftshijt for your great contributions!

**Pull request merged**